src/Storages/checkAndGetLiteralArgument.cpp

```diff
@@ -36,5 +36,5 @@ template UInt64 checkAndGetLiteralArgument(const ASTPtr &, const String &);
 template UInt8 checkAndGetLiteralArgument(const ASTPtr &, const String &);
 template bool checkAndGetLiteralArgument(const ASTPtr &, const String &);
 template String checkAndGetLiteralArgument(const ASTLiteral &, const String &);
-
+template UInt64 checkAndGetLiteralArgument(const ASTLiteral &, const String &);
```

Comment:
Looks unrelated

Reply:
Also compile error on s390x, invalid conversion on this following two lines.

```./build/./src/TableFunctions/TableFunctionGenerateRandom.cpp:66: undefined reference to `unsigned long DB::checkAndGetLiteralArgument<unsigned long>(DB::ASTLiteral const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&)'
/usr/bin/s390x-linux-gnu-ld: src/libdbms.a(StorageGenerateRandom.cpp.o): in function `operator()':
./build/./src/Storages/StorageGenerateRandom.cpp:477: undefined reference to `unsigned long DB::checkAndGetLiteralArgument<unsigned long>(DB::ASTLiteral const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&)'```

Reply:
Hm, okay